pydarn/plotting/fan.py

```diff
@@ -260,6 +251,9 @@ def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
                 alpha controls the transparency of
                 the fov color
                 Default: 0.5
+            kawrgs: key = value
+                Additional keyword arguments to be passed to axis_polar, e.g. lowlat, 
```

Comment:
Are there more options? I only see 2 this will need to be well documented for the user to know.